.github/workflows/test.yml

```diff
@@ -4,12 +4,12 @@ on: [push, pull_request]
 
 jobs:
   test:
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
```

Comment:
Could you revert this change? It is usually better to use a fixed version than just silently being upgraded. I'm also fine with updating this to `ubuntu-20.04`.

Reply:
using 20.04